src/System.Data.OleDb/tests/ReaderTests.cs

```diff
@@ -0,0 +1,143 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Data.OleDb;
+
+namespace System.Data.OleDb.Tests
+{
+    public class ReaderTests : IntegrationTestBase
+    {
+        [Fact]
+        public void EmptyReader()
+        {
+            command.CommandText =
```

Comment:
You'll want to include a txt file explaining how to set up the DB to run these tests